Services/DataBaseService/BrokerConsumers/FindConsumer.cs

```diff
@@ -0,0 +1,32 @@
+﻿using DataBaseService.Repositories.Interfaces;
+using DTO;
+using DTO.BrokerRequests;
+using MassTransit;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DataBaseService.BrokerConsumers
+{
+    public class FindConsumer : IConsumer<InternalFindUserRequest>
+    {
+        private readonly IUserRepository userRepository;
+
+        private UserCredential GetUserCredentialById(Guid Id)
+        {
+            return userRepository.GetUserCredentialById(Id);
+        }
+
+        public FindConsumer([FromServices] IUserRepository repository)
```

Comment:
Why is the variable name not a userRepository

Reply:
For what? Are there any other repositories?

Reply:
In constructor we explicitly specify the type of object. 
I named the field so as not confuse what type of repository it is